File: Desktop/Java/Arithmetic.java
import java.util.Scanner;
public class Arithmetic {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.println("Enter first number: ");
int number1 = input.nextInt(); 

System.out.println("Enter second number: ");
int number2 = input.nextInt();

System.out.println("Enter third number: ");
int number3 = input.nextInt();

int sum = number1 + number2 + number3;
int average = sum / 3;
int product = number1 * number2 * number3;
int largest = Math.max(Math.max(number1, number2), number3);
int smallest= Math.min(Math.min(number1, number2), number3);

System.out.println("Sum of the numbers:" + sum );
System.out.println("Average of the numbers:" + average);
System.out.println("Product of the numbers:" + product);
System.out.println("The largest number: " + largest);
System.out.println("The smallest number: " + smallest);
}


}

File: Desktop/Python/arithmetic.py
number1 = int(input("Enter the first integer:"))
number2 = int(input("Enter the second integer:"))
number3 = int(input("Enter the third integer:"))

sum_of_numbers = number1 + number2 + number3
average_of_numbers = sum_of_numbers / 3
product_of_numbers = number1 * number2 * number3

smallest_number = min(number1, number2, number3)
largest_number = max(number1, number2, number3)

print(sum_of_numbers, "is the sum of the numbers:")
print(average_of_numbers, "is the average of the numbers:")
print(product_of_numbers, "is the product of the numbers:")
print(smallest_number, "is the smallest number:")
print(largest_number, "is the largest number:")